src/core_plugins/elasticsearch/lib/clusters.js

```diff
@@ -0,0 +1,14 @@
+import Cluster from './cluster';
+
+const clusters = new Map();
```

Comment:
Is this something we can store in the runtime state of the elasticsearch plugin rather than hardcoding it into the module context?  The current implementation, in practical terms, is really just a global singleton to the entire process.